.github/workflows/bcny-ci.yml

```diff
@@ -15,26 +15,52 @@ jobs:
 
       matrix:
         include:
-          - arch: 'amd64'
+          - arch: 'AMD64'
             platform: 'x64'
-          - arch: 'arm64'
+          - arch: 'ARM64'
             platform: 'ARM64'
 
     steps:
       - uses: actions/checkout@v4
+        with:
+          submodules: "recursive"
 
       - uses: compnerd/gha-setup-vsdevenv@main
         with:
           host_arch: amd64
           arch: ${{ matrix.arch }}
 
+      - name: Build Workaround
+        run: |
+          git config --global user.email "builder@thebrowser.company"
+          git config --global user.name "Arc Builder"
+          git fetch origin pull/92/head
+          git cherry-pick FETCH_HEAD
+          git fetch origin pull/93/head
+          git cherry-pick FETCH_HEAD
+        working-directory: external/crashpad
+
       - name: Configure Sentry
         run: >
-          cmake -B out
-                -D CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION=10.0.19041.0
-                -G "Visual Studio 17 2022"
-                -A ${{ matrix.platform }}
-                -D CMAKE_SYSTEM_NAME=Windows
-                -D CMAKE_SYSTEM_PROCESSOR=${{ matrix.arch }}
+          cmake -B out `
+                -D CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION=10.0.19041.0 `
+                -G "Visual Studio 17 2022" `
+                -A ${{ matrix.platform }} `
+                -D CMAKE_SYSTEM_NAME=Windows `
+                -D CMAKE_SYSTEM_PROCESSOR=${{ matrix.arch }} `
       - name: Build Sentry
         run: cmake --build out --config Release
+
+      - name: Stage Sentry
+        run: |
+          New-Item -ItemType Directory -Path sentry-native\bin\win64 -Force | Out-Null
+          New-Item -ItemType Directory -Path sentry-native\include -Force | Out-Null
+          New-Item -ItemType Directory -Path sentry-native\lib\win64 -Force | Out-Null
+          Copy-Item include\sentry.h sentry-native\include\
+          Copy-Item out\Release\sentry.dll sentry-native\bin\win64\
+          Copy-Item out\Release\sentry.lib sentry-native\lib\win64\
```

Comment:
It looks like these archives are missing a few files:
- `bin\crashpad_handler.exe`
- `bin\crashpad_handler.pbd`
- `bin\crashpad_wer.dll`
- `bin\crashpad_wer.pdb`
- `bin\sentry.pdb`

I assume these are getting generated, we just need to include them

Reply:
Yeah, this is due to lack of knowledge of what was actually needed and trying to follow the instructions in swift-sentry.  Thanks for identifying the missing pieces!

Reply:
Done!